backend/queues/makeOfferProcessor.js

```diff
@@ -34,48 +57,126 @@ async function processor(job) {
 
   const { shopId, encryptedData, paymentCode } = job.data
   log.info(`Creating offer for shop ${shopId}`)
+  let confirmation
 
   try {
-    const shop = await getShop(shopId)
+    const shop = await Shop.findOne({ where: { id: shopId } })
+    if (!shop) {
+      throw new Error(`Failed loading shop with id ${shopId}`)
+    }
+
     queueLog(5, 'Load encrypted shop config')
-    const network = await getNetwork(shop.networkId)
+    const network = await _getNetwork(shop.networkId)
     const networkConfig = encConf.getConfig(network.config)
     const shopConfig = encConf.getConfig(shop.config)
 
     queueLog(10, 'Creating offer')
     const lid = ListingID.fromFQLID(shop.listingId)
-    const offer = createOfferJson(lid, encryptedData, paymentCode)
-    const ires = await postOfferIPFS(network, offer)
+    const offer = _createOfferJson(lid, encryptedData, paymentCode)
+    const ipfsHash = await _postOfferIPFS(network, offer)
```

Comment:
What's with underscore-prefixing all these?

Reply:
This is to denote them as private. I think it is a fairly common convention? but happy to change if that's unorthodox.

Reply:
Generally think it's private in classes but I really don't care that much.  Fine to leave as is.